Gemfile

```diff
@@ -10,41 +10,8 @@ group :development, :test do
   gem "simplecov", require: false
 end
 
-# BEGIN ENGINE_CART BLOCK
```

Comment:
This should always be in the git repo. The way you use engine cart is to run the installer, and then check in the results.